### ISSUE TITLE
missing 'babel-core' and typo 

### ISSUE THREAD

**Issue by oliver41** [treehouse-projects/webpack-workshop#1]
Hello, i think there are 2 Problems

- for me it is not working without installing 'babel-core'

- the js file tries to import ./loadDropDown but the js file is named /loadDrop**d**own.js
 import loadDropDown from "./loadDropDown";

**Comment by chris-gaona** [comment]
Yup it wasn't working for me either until 'babel-core' was installed. The following stackoverflow was kind of helpful explaining why: http://stackoverflow.com/questions/35215461/error-in-cannot-find-module-babel-core-using-react-js-webpack-and-express-s
